README.md

```diff
@@ -162,4 +162,79 @@ and define the following in your `csproj`:
 These attributes will ask for the runtime test engine to replace the ones defined by the `Uno.UI.RuntimeTests.Engine` package.
 
 ## Running the tests automatically during CI
```

Comment:
We should add a note about using Uno.UITest to run the runtime tests for Android/iOS/WASM as well, even if it is hopefully temporary. We can point to current apps that do this currently

Reply:
Added here https://github.com/unoplatform/uno.ui.runtimetests.engine/pull/156/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R242